granulate_utils/linux/cgroups/base_cgroup.py

```diff
@@ -55,6 +56,10 @@ def cgroup_mount_path(self) -> Path:
     def cgroup_path(self) -> Path:
         return Path(self.get_cgroup_hierarchies()[self.subsystem])
 
+    @property
+    def pids_in_cgroup(self) -> List[int]:
+        return [int(proc) for proc in self.read_from_control_file(self.cgroup_procs).split()]
```

Comment:
Return a `Set` instead to remove dups?

![image](https://user-images.githubusercontent.com/3260178/201529382-2b7c87a1-aa93-4912-a39c-a8cb0c8a156f.png)

Reply:
Great catch.